dist/notify.js

```diff
@@ -202,4 +207,6 @@
 
   exports['default'] = Notify;
 
+  Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Are you using a different version of jspm to the version that's in `package.json` here? When I pull down your branch locally and run `npm run build` this doesn't seem to get generated.